packages/app-project/pages/_app.js

```diff
@@ -69,11 +70,12 @@ export default class MyApp extends App {
 
   render () {
     const { Component, pageProps, theme } = this.props
+    const mergedThemes = _.merge({}, base, theme)
```

Comment:
Should this happen here, or inside the grommet theme?

Reply:
I think here. We should leave it to the consumer to decide which theme to merge which since Grommet core ships with a few? FWIW, I don't think we should have to do this at all because the top level `Grommet` component does a deep merge, but the Grommet devs recommended it when I bought up the merging issues in slack.